clients/src/main/java/org/apache/kafka/clients/InFlightRequests.java

```diff
@@ -148,4 +151,10 @@ public int inFlightRequestCount() {
 
         return nodeIds;
     }
+    
+    public interface InFlightRequest {
```

Comment:
Is it necessary to make this an interface? It seems there is only one implementation.